Practice futurespective (#711)
vmware-tanzu/tanzu-dev-portal: content/practices/futurespective/index.md

---
title: '"Sailboat" Futurespective'
linkTitle: "Futurespective"

> description: "A type of retrospective where you start from the goal to find ways on how to get there. Teams place themselves in the future by imagining that their goal has been reached. By exploring their imaginary past, teams agree how to work together to reach the goal."

Author: ```suggestion
description: "Using the "sailboat" retrospective format, start from the goal to find ways on how to get there. Teams place themselves in the future by imagining that their goal has been reached. By exploring their imaginary past, teams agree how to work together to reach the goal."
```

> description: "Using the "sailboat" retrospective format, start from the goal to find ways on how to get there. Teams place themselves in the future by imagining that their goal has been reached. By exploring their imaginary past, teams agree how to work together to reach the goal."

Author: ```suggestion
description: "Using the \"sailboat\" retrospective format, start from the goal to find ways on how to get there. Teams place themselves in the future by imagining that their goal has been reached. By exploring their imaginary past, teams agree how to work together to reach the goal."
```
